src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -415,7 +415,7 @@ public virtual void Write(TextWriter writer, TextSpan span, CancellationToken ca
             }
         }
 
-        internal ImmutableArray<byte> GetChecksum(bool useDefaultEncodingIfNull = false)
+        public ImmutableArray<byte> GetChecksum(bool useDefaultEncodingIfNull = false)
```

Comment:
actually, then, will there ever be someone who use it with useDefaultEncodingIfNull == false?